### PR TITLE
Add validator endpoint to REST API

### PR DESCRIPTION
**Pull request by danlamanna**
This adds the capability to query Romanesco for the validators it has loaded, optionally filtering by the type and format.

### CODE REVIEW

File: server/__init__.py [hunk 1]
```diff
@@ -363,5 +364,7 @@ def romanescoStopRun(jobId, params):
         (':itemId', 'romanesco', ':jobId'),
         romanescoStopRun)
 
+    info['apiRoot'].validator = Validator(getCeleryApp())
```
Comment (zachmullen) [inline server/__init__.py]: I think we should namespace this better, e.g. `romanesco_validator`.

**Comment by zachmullen** [comment]
This change has some places that need PEP8 compliance, hopefully the failing tests will show those places.

**Comment by danlamanna** [comment]
@zachmullen They should be fixed now.

**Comment by zachmullen** [comment]
LGTM

**Pull request merged**